src/main/java/com/zufar/onlinestore/cart/entity/ShoppingSession.java

```diff
@@ -33,7 +34,7 @@ public class ShoppingSession {
     @Column(name = "user_id", nullable = false)
     private UUID userId;
 
-    @OneToMany(mappedBy = "shoppingSession", cascade = CascadeType.ALL)
+    @OneToMany(mappedBy = "shoppingSession", cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
avoid cascade type all

Reply:
Fixed
Added //TODO to think about it better